modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -15,88 +14,56 @@ The network attachment definition must be in the same namespace as the pod or vi
 
 .Procedure
 
-. Create a new file for the network attachment definition in any local directory.
-The file must have the following contents, modified to match your
-configuration:
+. Create a network attachment definition manifest.
+The manifest must have the following contents, modified to match your configuration:
 +
 [source,yaml]
 ----
 apiVersion: "k8s.cni.cncf.io/v1"
 kind: NetworkAttachmentDefinition
 metadata:
-  name: a-bridge-network
+  name: <a-bridge-network> <1>
   annotations:
-    k8s.v1.cni.cncf.io/resourceName: bridge.network.kubevirt.io/br0 <1>
+    k8s.v1.cni.cncf.io/resourceName: bridge.network.kubevirt.io/<br0> <2>
```

Comment:
Hmm. Even though it's replaceable, I wouldn't put `br0` into angled brackets since it is an actual bridge name. I'd use something like `<bridge-interface>` instead.